src/api/apiSubjects.js

```diff
@@ -0,0 +1,92 @@
+import axios from "axios";
+
+const BASE_URL = "https://openmind-api.vercel.app/14-5";
+const headers = { "Content-Type": "application/json; charset=utf-8" };
```

Comment:
axios instance를 만들어서 중복되는 코드들을 줄여보면 좋을것 같아요!


```suggestion
import Axios from "axios";

const BASE_URL = "https://openmind-api.vercel.app/14-5";
const headers = { "Content-Type": "application/json; charset=utf-8" };
const axios = Axios.create({
  baseURL: BASE_URL,
  headers,
});
```

_<사용예시>_
```javascript
export async function getAllSubjects() {
  try {
    const res = await axios.get("/subjects");
    return res.data;
  } catch {
    throw new Error("에러가 발생했습니다.");
  }
}
```

Ref. https://axios-http.com/docs/instance